actions/common/search_methods.go

```diff
@@ -0,0 +1,54 @@
+package common
+
+import (
+	"math"
+
+	"github.com/bitcoin-sv/spv-wallet/engine/datastore"
+)
+
+// Count object to use when returning a count of database query results
+type Count struct {
+	Content any  `json:"content"`
```

Comment:
It would be good to use generics (like in ConditionsModel struct) instead of any.

Reply:
Done